packages/shared/src/models/bootc.ts

```diff
@@ -29,3 +29,5 @@ export interface BootcBuildInfo {
 }
 
 export type BootcBuildStatus = 'running' | 'creating' | 'success' | 'error' | 'lost' | 'deleting';
+
+export const BOOTC_BUILD_LOG_FILENAME = 'image-build.log';
```

Comment:
Leftover from previous commit? I thought we removed this and is not being used by frontend.